cmd/tsbs_generate_queries/databases/mongo/devops-naive.go

```diff
@@ -42,43 +44,46 @@ func (d *NaiveDevops) GroupByTime(qi query.Query, nHosts, numMetrics int, timeRa
 	metrics, err := devops.GetCPUMetricsSlice(numMetrics)
 	panicIfErr(err)
 
-	bucketNano := time.Minute.Nanoseconds()
 	pipelineQuery := []bson.M{
 		{
-			"$match": map[string]interface{}{
+			"$match": bson.M{
 				"measurement": "cpu",
-				"timestamp_ns": map[string]interface{}{
-					"$gte": interval.StartUnixNano(),
-					"$lt":  interval.EndUnixNano(),
+				"time": bson.M{
+					"$gte": interval.Start(),
```

Comment:
Previously the time field was stored in nanoseconds, but this was changed for the time-series backed configuration. `interval.Start()` returns a `time.Time` object, which can be serialized to BSON type Date.

Reply:
Helpful comment, thanks!